scss/widgets/base/_toolbar.scss

```diff
@@ -182,35 +210,3 @@
 .dx-toolbar-item-invisible {
   display: none;
 }
-
-.dx-toolbar-hidden-button-group {
-  &.dx-state-hover {
-    background-color: transparent;
-  }
-
-  .dx-list-item-content {
```

Comment:
is it ok that this styles are removed at all?

Reply:
It is ok. The test-cafe screenshots approve it. If I right remember this styles are redundant for new hierarchy